phoenix-core/src/it/java/org/apache/phoenix/end2end/DescOrderKeysIT.java

```diff
@@ -439,5 +439,121 @@ public void testDecimalDescOrderMultiplePKs() throws Exception {
     }
   }
 
+  @Test
```

Comment:
Good idea to test with various data types.
Also tests for order by clause on that field.

Reply:
Done

Reply:
Looking good, do you want to add a test for order by clause on the key?

Reply:
done in the latest revision